backend/clubs/serializers.py

```diff
@@ -1531,6 +1532,19 @@ class Meta(UserClubVisitSerializer.Meta):
         pass
 
 
+class SearchQuerySerializer(serializers.ModelSerializer):
+    """
+    Used by users to enter search result into backend.
+    """
+
+    person = serializers.HiddenField(default=serializers.CurrentUserDefault())
+    query = serializers.CharField()
```

Comment:
There is no need to define this if you're not going to customize the default behavior.